FHEM/lib/SD_ProtocolData.pm

```diff
@@ -121,9 +114,9 @@ package lib::SD_ProtocolData;
 				paddingbits			=> '8',
 			},
 		"0.2"	=>	## other Sensors | for sensors how tol is runaway (260+tol | 9650)
-							# MS;P1=-2140;P2=309;P3=-4690;P4=-9695;D=2421232323232121232123232321212121212121212123212121232121;CP=2;SP=4;R=211;m1;
-							# MS;P0=-9703;P1=304;P2=-2133;P3=-4689;D=1012131312131212131213131312121212121212121212131312131212;CP=1;SP=0;R=208;
-							# MS;P0=138;P1=-2140;P2=315;P3=-9704;P4=-4713;P5=234;D=2321212421242454210424212421512121215121512124212121542121;CP=2;SP=3;R=210;
+							# Mebus | Id:151 Ch:1 T: 0.4 Bat:low   MS;P1=-2140;P2=309;P3=-4690;P4=-9695;D=2421232323232121232123232321212121212121212123212121232121;CP=2;SP=4;R=211;m1;
+							# Mebus | Id:151 Ch:1 T: 0.3 Bat:low   MS;P0=-9703;P1=304;P2=-2133;P3=-4689;D=1012131312131212131213131312121212121212121212131312131212;CP=1;SP=0;R=208;
+							# no decode!                           MS;P0=138;P1=-2140;P2=315;P3=-9704;P4=-4713;P5=234;D=2321212421242454210424212421512121215121512124212121542121;CP=2;SP=3;R=210;
```

Comment:
welchen Sinn macht es kaputte Nachrichten drin zu lassen?

Reply:
Darüber wird informiert in der PR Beschreibung. - No Decode - darüber MUSS entschieden werden!

Das die komplette Überarbeitung erstmal reinkommt ist mit dem aprove gut.

Wie wir damit verfahren, müssen wir klären und ggf auf die Suche gehen ob es falsche Nachrichten sind oder wieso...

Den Konflikt für den PR behebe ich morgen damit die Überarbeitung übernommen werden kann.

Mobil, frohe Ostern Euch allen!